hero_trump/card.py

```diff
@@ -26,23 +31,24 @@ def __init__(
         self.tools = tools
         self.present = present
 
+    def __str__(self):
+        return (self.name)
+
+    def __repr__(self):
+        return str(self.name)
+
+    def assign_card(self, row):
```

Comment:
Can you add assign_card() to the init and take the row as an arg in the init, then the card is properly initialised.